monitor/parent/entrypoint.sh

```diff
@@ -38,9 +38,22 @@ if mountpoint -q /etc/netdata && [ -z "$(ls -A /etc/netdata)" ]; then
   cp -a /etc/netdata.stock/. /etc/netdata
 fi
 
+# Setup cron job with environment variables
```

Comment:
Why setting up the cron before running initial stream whitelist update?

To me it looks like it gives a (bad) opportunity to have both cron + initial run in parallel should cron trigger very fast. Very unlikely, but very nasty to debug. If there is no other reasons, I would prefer to run initial update then setup cron.